source/Calamari.Tests/Fixtures/Manifest/ExecuteManifestCommandFixture.cs

```diff
@@ -94,8 +92,7 @@ CalamariResult ExecuteCommand(VariableDictionary variables, string extensions =
                 return Invoke(Calamari()
                               .Action("execute-manifest")
                               .Argument("variables", variablesFile.FilePath)
-                              .Argument("sensitiveVariablesPassword", "GB8KdBqYRlgAON9ISUPdnQ==")
-                              .Argument("extensions", extensions));
```

Comment:
Turns out tests were using extensions to load the `Calamari.Tests` assembly so they could have test commands and stuff.. so I had to change some things around to get everything to work again.